app/src/main/java/com/framgia/wsm/data/source/remote/api/service/WSMApi.java

```diff
@@ -3,25 +3,31 @@
 import com.framgia.wsm.data.model.Request;
 import com.framgia.wsm.data.model.RequestOff;
 import com.framgia.wsm.data.model.RequestOverTime;
+import com.framgia.wsm.data.model.User;
 import com.framgia.wsm.data.source.remote.api.request.SignInRequest;
 import com.framgia.wsm.data.source.remote.api.response.BaseResponse;
 import com.framgia.wsm.data.source.remote.api.response.SignInDataResponse;
 import com.framgia.wsm.data.source.remote.api.response.TimeSheetResponse;
+import com.framgia.wsm.data.source.remote.api.response.UserProfileResponse;
 import io.reactivex.Observable;
 import java.util.List;
 import retrofit2.http.Body;
 import retrofit2.http.GET;
 import retrofit2.http.POST;
+import retrofit2.http.Path;
 import retrofit2.http.Query;
 
 /**
  * Created by le.quang.dao on 10/03/2017.
  */
 
 public interface WSMApi {
-    @POST("/api/v1/sign_in")
+    @POST("/api/sign_in")
     Observable<BaseResponse<SignInDataResponse>> login(@Body SignInRequest signInRequest);
 
+    @GET("/api/dashboard/users/{user_id}")
+    Observable<BaseResponse<UserProfileResponse>> getUserProfile(@Path("user_id") int userId);
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "user_id" appears 4 times in this file; the first occurrence is on line 29
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals